File: system/src/main/java/com/cys/system/common/service/TypeTemplateService.java
package com.cys.system.common.service;

import com.cys.system.common.common.pojo.Result;
import com.cys.system.common.pojo.Brand;
import com.cys.system.common.pojo.TypeTemplate;

import java.util.List;
import java.util.Map;

public interface TypeTemplateService {

    Result add(TypeTemplate typeTemplate);

    Result update(TypeTemplate typeTemplate);

    Result findOne(Integer id);

    Result delete(Long[] ids);

    Result listTypeTemplate(TypeTemplate typeTemplate, Integer page, Integer rows);

    List<Map> selectOptionList(Integer shopId);

    List<Map> selectBrandList();
}

File: system/src/main/resources/static/system/js/service/typeTemplateService.js
//服务层
app.service('typeTemplateService',function($http){
	//查询实体
	this.findOne=function(id){
		return $http.get(basePath+"/typeTemplate/findOne?id="+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post(basePath+"/typeTemplate/add",entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.put(basePath+"/typeTemplate/update",entity );
	}
	//删除
	this.dele=function(ids){
		return $http.delete(basePath+"/typeTemplate/delete?ids="+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post(basePath+"/typeTemplate/search?page="+page+"&rows="+rows, searchEntity);
	}
    //读取列表数据绑定到表单中
    this.selectBrandList=function(){
        return $http.get(basePath+"/typeTemplate/selectBrandList");
    }
	this.selectOptionList=function(){
		return $http.get(basePath+"/typeTemplate/selectOptionList");
	}
});
